tests/Fixture/LogsFixture.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         1.2.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Test\Fixture;
+
+use Cake\TestSuite\Fixture\TestFixture;
+
+/**
+ * Short description for class.
+ *
+ */
+class LogsFixture extends TestFixture
```

Comment:
We have many existing fixtures. Quite a few have nullable fields. For example the `AuthorsFixtures` has a nullable `name` column. The `UsersFixtures` has a nullable `username` column. Could we re-use an existing fixture instead of adding a new one? Having more fixtures slows down the test suite and increases maintenance costs.